handlers/songs.go

```diff
@@ -0,0 +1,32 @@
+package handlers
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/gobridge/gopher/bot"
+)
+
+// Songs inspects the message for Spotify, Soundcloud, Tidal links.
+func Songs() bot.Handler {
+	var (
```

Comment:
Please move this outside of the function to package level variables.
There isn't a need to MustCompile these each time Songs() is called.

Reply:
These are only compiled once. The function returns a closure that captures the regex.

Reply:
Okay. I do think that could panic should happen at startup rather than in a package's exported function.

Reply:
That's true. It may be better to have the function return an error or simply do as you suggested and move them to package scope. Part of the reason I didn't put too much though into it is that I think this functionality is a good candidate for removal, but that's not a very satisfying excuse.

Reply:
SGTM. I merged the PR, so any changes will be in a follow up. :-)